src/codegen/templates/property_getter_template.py.mako

```diff
@@ -1,13 +1,21 @@
 <%def name="script_property_getter(attribute)">\
 <%
         from codegen.utilities.text_wrappers import docstring_wrap
-        from codegen.utilities.attribute_helpers import get_generic_attribute_function_name, get_generic_attribute_function_type
+        from codegen.utilities.attribute_helpers import get_generic_attribute_function_name, get_generic_attribute_function_type, ATTRIBUTES_RETURN_FILE_PATH_TYPE
     %>\
+    %if attribute.name in ATTRIBUTES_RETURN_FILE_PATH_TYPE:
```

Comment:
FYI, I was going to suggest that you wait until https://github.com/ni/nidaqmx-python/issues/209 so we don't end up adding special cases, but you know what? It's fine. The code for returning the value has a ton of special cases anyway. In the worst case, the code declaring the property would have the same number of special cases, but it will probably have less. 😅